custom-login/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "2.0.0",
   "private": true,
   "scripts": {
-    "start": "vue-cli-service serve",
+    "start": "node server.js",
```

Comment:
How about keep this command unchanged, but add the server as another layer just for heruko deployment? 

The idea is to still let devs stick with vue cli ecosystem when they are trying to play with the code, then they can decide wether to deploy it as a SPA or web app.